src/resources/js/tribe-events-ajax-calendar.js

```diff
@@ -142,37 +142,37 @@
 
 		}
 
-		function tribe_mobile_setup_day( date, date_name ) {
-
-			var $container = $wrapper.find( '#tribe-mobile-container' ),
-				$day_blocks = $wrapper.find( '.tribe-mobile-day' ),
-				$mobile_trigger = $wrapper.find( '.mobile-trigger' ),
-				$target_day = $wrapper.find( '.tribe-mobile-day[data-day="' + date + '"]' ),
-				full_date_name = 'undefined' === typeof( date_name )  ? '' : date_name,
-				day_data = { "date": date, "date_name": full_date_name };
-
-			$mobile_trigger
-				.removeClass( 'mobile-active' );
-
-			// If full_date_name is empty then default to highlighting the first day of the current month
-			var filter = full_date_name.length
-				? '[data-date-name="' + full_date_name + '"]'
-				: '.tribe-events-thismonth[data-day="' + date + '"]';
+		function tribe_mobile_setup_day( $date ) {
```

Comment:
I like the parameters in this updated function. I was originally curious about backward compatibility here, but having read through the code and seeing how/when this is called, I'm doubtful that we'd be impacting customizations by folks. Thoughts?